CMake/macosMaxAllowd.c

```diff
@@ -0,0 +1,8 @@
+#include <AvailabilityMacros.h>
+#include <stdio.h>
+
+int main(int argc, char **argv) {
+  unsigned int ver = __MAC_OS_X_VERSION_MAX_ALLOWED;
+  printf("%d.%d.%d", ver/10000, (ver/100)%100, ver%100);
+  return 0;
+}
```

Comment:
If this is going to be used, please rename this to `CMake/macosMaxAllowed.c` (fix spelling).